jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -208,6 +208,7 @@ public class StartArgs
     private boolean listConfig = false;
     private boolean version = false;
     private boolean dryRun = false;
+    private boolean multiLine = false;
```

Comment:
This should be a separate concept, not as a "part" or "section" on `--dry-run=<parts>`

Consider instead a `--dry-run-format=<type>` where we have a few options.

* `sh` - what we do now
* `windows` - an alternate mode for windows
* `newline` - what this `multiLine` is doing

And in the future we might even be able to support the various Daemon tooling and produce JSON or XML of the args that those tools consume.

Reply:
But he do you select which parts to output with this new option?

Reply:
The existing `--dry-run=<parts>` still exists.
Select what you want from that.

How the parts are presented are impacted by the new arg.

default behavior: (as `--dry-run-format=sh` is default, that's what is used)

```
$ java -jar start.jar --dry-run=opts,path,main,args
```

or on windows

```
> java -jar start.jar --dry-run=opts,args --dry-run-format=windows
```

or for new lines

```
$ java -jar start.jar --dry-run=opts,path,main,args --dry-run-format=newline
```

and in the future

```
$ java -jar start.jar --dry-run=opts,path,main,args --dry-run-format=json
```

Reply:
This is too complex and verbose for a minor feature